Use /var/log/mythtv instead of just /var/log for mythtv logs.
==========================================================================
--- a/programs/mythbackend/housekeeper.cpp
+++ b/programs/mythbackend/housekeeper.cpp
@@ -324,7 +324,7 @@
                                           "mythfilldatabase");
     QString mfarg = gContext->GetSetting("MythFillDatabaseArgs", "");
     QString mflog = gContext->GetSetting("MythFillDatabaseLog",
-                                         "/var/log/mythfilldatabase.log");
+                                         "/var/log/mythtv/mythfilldatabase.log");
 
     if (mfpath == "mythfilldatabase")
         mfpath = GetInstallPrefix() + "/bin/mythfilldatabase";
--- a/programs/mythwelcome/welcomedialog.cpp
+++ b/programs/mythwelcome/welcomedialog.cpp
@@ -460,7 +460,7 @@
                                           "mythfilldatabase");
     QString mfarg = gContext->GetSetting("MythFillDatabaseArgs", "");
     QString mflog = gContext->GetSetting("MythFillDatabaseLog",
-                                         "/var/log/mythfilldatabase.log");
+                                         "/var/log/mythtv/mythfilldatabase.log");
 
     if (mflog.isEmpty())
         command = QString("%1 %2").arg(mfpath).arg(mfarg);
